src/app/pages/Homescreen/Homescreen.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import HomescreenLink from "../../components/HomescreenLink/HomescreenLink";
+import styles from "./Homescreen.module.css";
+import HomescreenElement from "./HomescreenElement";
+
+function Homescreen(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <main>
+        <img className={styles.logo} src="./logoWithText.png" alt="" />
+      </main>
+      <div className={styles.bottom}>
+        <HomescreenLink text={"Planen"} link={""} />
+        <HomescreenLink text={"Zufallsgenerator"} link={""} />
+        <HomescreenElement />
+      </div>
+    </div>
```

Comment:
why did you change the footer into a div? a footer seems right here and is more specific.

Reply:
I thought if I also work without a header, I prefer to stay strict and also do not use a footer :)